tests/test_validators_dataclass.py

```diff
@@ -180,7 +179,7 @@ def root_validator(cls, values):
     assert exc_info.value.errors() == [
         {
             'ctx': {'error': 'foobar'},
-            'input': HasRepr(IsStr(regex=re.escape("ArgsKwargs((1,), {'b': 'snap dragon'})"))),
+            'input': HasRepr("ArgsKwargs((1,), {'b': 'snap dragon'})"),
```

Comment:
I added this test the other day, but while working on this I realized I didn't need the `IsStr(regex=` thing since I was just doing an exact equality check. So I removed here.